File: Selnium script for testing/Script with Python/test_scenario_26.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_26():
    """
    Dummy test run for scenario 26
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_26")

    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_124")
    time.sleep(2)
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    assert "Dashboard" in driver.title
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    assert "Dashboard" in driver.title
    driver.find_element(By.LINK_TEXT, "Log Out").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_824")
    driver.find_element(By.NAME, "password_field").send_keys("test_data_962")
    time.sleep(2)
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    time.sleep(2)

    driver.quit()

if __name__ == "__main__":
    execute_test_26()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario26.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario26 {
    public static void executeTest26() {
        /*
         * Dummy test run for scenario 26
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_26");

        try {
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.linkText("Log Out")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            Thread.sleep(2000);
            if (!driver.findElement(By.name("password_field")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.name("password_field")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_472");
            driver.findElement(By.id("checkout_btn")).click();
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_325");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.cssSelector(".promo-code")).click();
            if (!driver.findElement(By.linkText("Log Out")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest26();
    }
}
